### PR TITLE
Refactoring UI-KIT class names

### PR DESCRIPTION
**Pull request by piall**
Previously lots of code were written already for class names with {}. So, other contributors who contributed to `uiKit.js` also followed the same style which was not necessary.
In my last PR, Ms. Olga suggested refactoring all the `class names` by replacing {} with " ", where **{}** are not necessary.

reference( https://github.com/hackforla/VRMS/pull/557#discussion_r633975937 )

### CODE REVIEW

**Review by drubgrubby — APPROVED** [review]
@mdPial - These changes look good.  Thanks for cleaning this up. 
(I'm going to leave this here for the day in case anyone else wants to review before merging).

**Comment by piall** [comment]
@drubgrubby Thank you for self reviewing

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 1]
```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
...
         <p>Text content</p>
-        <p className={'dev-comment'}>Main text: 'Open Sans', 14px, regular</p>
+        <p className="dev-comment">Main text: 'Open Sans', 14px, regular</p>
 
-        <p className={'italic'}>Text content</p>
-        <p className={'dev-comment'}>Main text: 'Open Sans', 14px, italic</p>
+        <p className="italic">Text content</p>
+        <p className="dev-comment">Main text: 'Open Sans', 14px, italic</p>
 
         <a key={'default-link'} href={'/'}>
```
Comment (averdin2) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: I noticed on line 72 that there are {} around the a tag's attributes like key and href. In my experience these also do not need the {} around the strings. This also occurs in various places in the code for a tags. You may want to discuss this with a more senior member.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 2]
```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
...
-        <Button content={`Sign In`} className={'btn-square'} />
-        <p className={'dev-comment'}>
-          Square button, use with class .btn-square
-        </p>
-        <button className={'btn-confirm'}>
-          <img src={confirmIcon} className={'btn-icon'} alt={'confirm'} />
+        <p className="dev-comment">Primary default button, use without class</p>
+        <Button content={`Sign In`} className="btn-accent" />
+        <p className="dev-comment">Accent button, use with class .btn-accent</p>
+        <Button content={`Sign In`} className="btn-square" />
```
Comment (averdin2) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: Where the Button component has content={\`Sign In\`}, I don't see any reason why there needs to be {} and \`\`. I think the Sign In should be a string with "" and no need for {} or \`\`. Again, perhaps ask another more senior dev about this.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 3]
```diff
@@ -153,120 +143,120 @@ const DevUiKit = () => {
...
+        <p className="dev-comment">Tooltip, works on hover</p>
       </div>
 
       {/*** INPUTS ***/}
-      <div className={'inputs-container'}>
-        <h1 className={'kit-sec-title'}>*** INPUTS ***</h1>
+      <div className="inputs-container">
+        <h1 className="kit-sec-title">*** INPUTS ***</h1>
 
         <Input placeholder={'Enter your email'} type={'email'} />
```
Comment (averdin2) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: In this case, the attributes of the Input component don't need {} in my opinion. Typically, {} is used when inputting a variable or some JS functionality. {} aren't needed to input strings.

**Review by averdin2 — COMMENTED** [review]
@mdPial I saw some other places other than className where it seems to me that {} aren't needed. I would ask a more senior dev on the team if my thinking is right, and perhaps make changes to these places as well.

**Comment by piall** [comment]
@OlgaBilogurova or @drubgrubby Would you like to review all comments of @averdin2?

**Comment by drubgrubby** [comment]
@mdPial -> I will review Alex's comments later today.

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 1 above)
Comment (drubgrubby) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: Agreed.  You don't need the curly braces here.

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 2 above)
Comment (drubgrubby) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: I agree, you don't need the curly braces here either.

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 3 above)
Comment (drubgrubby) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: I agree, again.

**Comment by drubgrubby** [comment]
@mdPial -> The original issue was for refactoring the class names, and you did to that correctly in your original PR.  But @averdin2  made a good catch on a couple of additional instances of unnecessary curly braces that you might as well remove while you're at it. 

I agree with Alex's comments; go ahead and make those changes.

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 1 above)
Comment (OlgaBilogurova) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: I agree.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 4]
```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
...
+        <a key={'create-link'} className="redirect-link accent-link" href={'/'}>
           CREATE ACCOUNT
         </a>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           Accent Link: 13px, bold, $accent-royal-blue-color
         </p>
 
         <a
           key={'tooltip-link'}
```
Comment (OlgaBilogurova) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: Could be replaced with: key="tooltip-link"

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 2 above)
Comment (OlgaBilogurova) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: I agree

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 5]
```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
...
-        </p>
-        <button className={'btn-confirm'}>
-          <img src={confirmIcon} className={'btn-icon'} alt={'confirm'} />
+        <p className="dev-comment">Primary default button, use without class</p>
+        <Button content={`Sign In`} className="btn-accent" />
+        <p className="dev-comment">Accent button, use with class .btn-accent</p>
+        <Button content={`Sign In`} className="btn-square" />
+        <p className="dev-comment">Square button, use with class .btn-square</p>
+        <button className="btn-confirm">
+          <img src={confirmIcon} className="btn-icon" alt={'confirm'} />
```
Comment (OlgaBilogurova) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: The same for alt here.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 6]
```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
...
           More details here
         </a>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           Tooltip Link: 10px, bold, $link-accent-color
         </p>
 
-        <a className={'join-link'} href={'/'}>
-          Join new project <span className={'join-plus'}>+</span>
+        <a className="join-link" href={'/'}>
```
Comment (OlgaBilogurova) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: The same for href.

File: client-mvp-04/src/utils/uiKit/uiKit.js (hunk 3 above)
Comment (OlgaBilogurova) [reply client-mvp-04/src/utils/uiKit/uiKit.js]: Agree.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 7]
```diff
@@ -153,120 +143,120 @@ const DevUiKit = () => {
...
+        <h1 className="kit-sec-title">*** INPUTS ***</h1>
 
         <Input placeholder={'Enter your email'} type={'email'} />
-        <p className={'dev-comment'}>Default Input, components/common/input</p>
+        <p className="dev-comment">Default Input, components/common/input</p>
 
-        <div className={'text-field-icon-container'}>
-          <img src={gitHubIcon} className={'text-field-icon'} alt={'gitHub'} />
+        <div className="text-field-icon-container">
+          <img src={gitHubIcon} className="text-field-icon" alt={'gitHub'} />
```
Comment (OlgaBilogurova) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: The same for alt.

File: client-mvp-04/src/utils/uiKit/uiKit.js [hunk 8]
```diff
@@ -153,120 +143,120 @@ const DevUiKit = () => {
...
 
-        <div className={'text-field-rounded-icon-container'}>
+        <div className="text-field-rounded-icon-container">
           <input
             type="text"
             name="text-icon-field"
             placeholder={`Search Reports`}
           />
-          <img src={searchIcon} className={'text-field-icon'} alt={'search'} />
+          <img src={searchIcon} className="text-field-icon" alt={'search'} />
```
Comment (OlgaBilogurova) [inline client-mvp-04/src/utils/uiKit/uiKit.js]: The same for alt.

**Review by OlgaBilogurova — CHANGES_REQUESTED** [review]
Hi @mdPial,

In all places where we pass string literals, curly braces could be removed. We just need them when passing props or variables.

Actually, both two JSX expressions are equivalent, but as for me without {} looks cleaner:
```
<MyComponent message="hello world" />
<MyComponent message={'hello world'} />
```

Please, take a look at the doc:
https://reactjs.org/docs/jsx-in-depth.html

**Comment by piall** [comment]
Thank you everyone for giving the time to review this PR :smile: . I have updated.

**Review by OlgaBilogurova — APPROVED** [review]

**Pull request merged**